app/controllers/v0/burial_claims_controller.rb

```diff
@@ -4,28 +4,36 @@ class BurialClaimsController < ApplicationController
     skip_before_action(:authenticate)
 
     def create
-      claim = SavedClaim::Burial.new(form: burial_claim_params[:form])
+      claim = claim_class.new(form: filtered_params[:form])
       unless claim.save
         validation_error = claim.errors.full_messages.join(', ')
-        log_message_to_sentry(validation_error, :error, {}, validation: 'burial_claim')
+        log_message_to_sentry(validation_error, :error, {}, validation: short_name)
 
         StatsD.increment("#{stats_key}.failure")
         raise Common::Exceptions::ValidationErrors, claim
       end
-
+      claim.process_attachments!
       StatsD.increment("#{stats_key}.success")
-      Rails.logger.info "ClaimID=#{claim.confirmation_number} Form=#{SavedClaim::Burial::FORM}"
+      Rails.logger.info "ClaimID=#{claim.confirmation_number} Form=#{claim.class::FORM}"
       render(json: claim)
     end
 
     private
 
-    def burial_claim_params
-      params.require(:burial_claim).permit(:form)
+    def short_name
+      'burial_claim'
+    end
+
+    def claim_class
+      SavedClaim::Burial
+    end
```

Comment:
I was really, really hoping to get this merged in last week and push off a refactor that made a shared base controller for Pension and Burial, but, here we are a week later.